tests/trestle/core/commands/author/component_test.py

```diff
@@ -119,10 +119,23 @@ def test_component_generate(tmp_trestle_dir: pathlib.Path, monkeypatch: MonkeyPa
     creation_time = assem_comp_path.stat().st_mtime
     assert model_utils.ModelUtils.models_are_equivalent(orig_component, assem_component, True)
 
+    # assemble again and confirm it is not written out since no change
     test_utils.execute_command_and_assert(assemble_cmd, CmdReturnCodes.SUCCESS.value, monkeypatch)
     assert creation_time == assem_comp_path.stat().st_mtime
 
-    ac1_path = tmp_trestle_dir / 'md_comp/comp_aa/comp_prof_aa/ac/ac-1.md'
+    # edit a rule param value
+    new_text = '      component-values:\n        - inserted value\n'
```

Comment:
could we also test setting multiple values?

Reply:
I think this would be a good learning task for Alejandro.  Can you sketch what is involved to him?  Otherwise I can do it Thurs.